services/wallet/api.go

```diff
@@ -84,7 +84,7 @@ func (api *API) GetBalancesByChain(ctx context.Context, chainIDs []uint64, addre
 	return api.s.tokenManager.GetBalancesByChain(ctx, clients, addresses, tokens)
 }
 
-func (api *API) FetchOrGetCachedWalletBalances(ctx context.Context, addresses []common.Address) (map[common.Address][]token.StorageToken, error) {
+func (api *API) FetchOrGetCachedWalletBalances(ctx context.Context, addresses []common.Address, forceRefresh bool) (map[common.Address][]token.StorageToken, error) {
```

Comment:
This PR is a good example of why it's important to use a request map instead of position args because then it would be possible to add a new struct field for `forceRefresh` and not break any client.

IMO it would be great if we followed this standard by the book because when endpoints are designed initially we cannot anticipate which other args it may need. I remember a handful of cases where such breaking changes were introduced in status-go PRs and could've been avoided. cc @igor-sirotin

Reply:
Totally agree @ilmotta, I didn't want to refactor the whole implementation, but a map would have made this change much simpler to handle for the clients and not breaking.